src/translation/Translation.ts

```diff
@@ -12,7 +12,7 @@ export class Translation {
     }
 
     get(key: string, parameters?: string[]): string {
-        var str: string = this.data[key]
+        let str: string = this.data[key]
```

Comment:
ups, I hope I learn this at some point :) (`let` sounds so arbitrary)

Reply:
Just use `const` 90% of the time :)